src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -245,6 +245,7 @@ static String getResource(String key) {
             {"R_statementPoolingCacheSizePropertyDescription",
                     "This setting specifies the size of the prepared statement cache for a connection. A value less than 1 means no cache."},
             {"R_gsscredentialPropertyDescription", "Impersonated GSS Credential to access SQL Server."},
+            {"R_msiObjectIdPropertyDescription", "Object Id of User Assigned Managed Identity to be used for generating access token for Azure AD MSI Authentication"},
```

Comment:
Id => ID? in both the entry name + description

Reply:
I would keep it camel case (msiClientId) to avoid confusions with other property cases.